stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/messages/MessageFooter.kt

```diff
@@ -69,7 +69,7 @@ public fun MessageFooter(
     Column(horizontalAlignment = alignment.contentAlignment) {
         MessageTranslatedLabel(messageItem)
         if (messageItem.showMessageFooter) {
-            var showEditLabel by remember { mutableStateOf(message.messageTextUpdatedAt != null) }
+            val showEditLabel = message.messageTextUpdatedAt != null
```

Comment:
The null value was being cached in the state, so when updated `message` was coming we were still using the cached value.